### DISCUSSION TITLE
Error: Attribute "Livewire\Attributes\Rule" must not be repeated

### DISCUSSION THREAD

**Discussion by viliusvsx** [livewire/livewire#5873] [7. Bugs (archived)] [answered]
Hello, 

I think I found a bug using multiple #[Rule] attributes on the same property. Based on documentation, I could use multiple rules attributes on a property, but when I do this, I get an error: `Attribute "Livewire\Attributes\Rule" must not be repeated`

```
#[Rule('required', message: 'Please enter a title.')]
#[Rule('min:5', message: 'Your title is too short.')]
public $title = '';
```

**Comment by nuernbergerA** [comment, marked as answer]
i made a PR https://github.com/livewire/livewire/pull/5876
